src/Model/Filter/Boolean.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Search\Model\Filter;
+
+class Boolean extends Base
+{
+
+    /**
+     * Default configuration.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'truthy' => [1, true, '1', 'true',  'yes', 'on'],
```

Comment:
I am not opposed on using consts to get closer to atoms (Elixir, Ruby, Python). However CakePHP is often not doing that.

It is something I'd like to see for 4.x or 5.x ... e.g. moving to only constants and methods and never hash assignments of options, as its ultra error prone and thus consumes **MEGATIME** (sorry ;p)